pkg/scheduler/admission/podnodeconstraints/admission.go

```diff
@@ -69,9 +69,12 @@ var resourcesToCheck = map[unversioned.GroupResource]unversioned.GroupKind{
 	kapi.Resource("podtemplates"):                               kapi.Kind("PodTemplate"),
 	kapi.Resource("replicationcontrollers"):                     kapi.Kind("ReplicationController"),
 	batch.Resource("jobs"):                                      batch.Kind("Job"),
+	batch.Resource("jobtemplates"):                              batch.Kind("JobTemplate"),
+	batch.Resource("scheduledjobs"):                             batch.Kind("ScheduledJob"),
 	extensions.Resource("deployments"):                          extensions.Kind("Deployment"),
 	extensions.Resource("replicasets"):                          extensions.Kind("ReplicaSet"),
 	extensions.Resource("jobs"):                                 extensions.Kind("Job"),
+	extensions.Resource("jobtemplates"):                         extensions.Kind("JobTemplate"),
```

Comment:
It's showing up in the reflective test.

On Thu, Jul 7, 2016 at 6:27 AM, Maciej Szulik <notifications@github.com>
wrote:

> In pkg/scheduler/admission/podnodeconstraints/admission.go
> <https://github.com/openshift/origin/pull/9578#discussion_r69884344>:
>
> >  	extensions.Resource("deployments"):                          extensions.Kind("Deployment"),
> >  	extensions.Resource("replicasets"):                          extensions.Kind("ReplicaSet"),
> >  	extensions.Resource("jobs"):                                 extensions.Kind("Job"),
> > +	extensions.Resource("jobtemplates"):                         extensions.Kind("JobTemplate"),
>
> JobTemplate exists only in batch, it was done as part of ScheduledJobs
> and is only in batch/v2alpha1.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/deea48f62fd9b32142d028658081cb0209812ecc..849a565571da716634dcbbb80b64363885fb8448#r69884344>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_px1dAs9C1RwqR4vNRUXGD-ZowoBDks5qTNSpgaJpZM4I-xTE>
> .
>